apps/web/app/(landing)/templates/data.ts

```diff
@@ -49,6 +49,17 @@ export const templates: Record<string, Template> = {
     language: "Typescript",
     framework: "Next.js",
   },
+  "nextjs-expiration": {
+    title: "Next.js Example with Temporary API Keys",
+    description:
+      "A simple Next.js app using Unkey to generate and verify API keys that expire after 60 seconds.",
+    authors: ["ethan-stone"],
+    repository: "https://github.com/unkeyed/unkey/tree/main/examples/nextjs",
```

Comment:
```suggestion
    repository: "https://github.com/unkeyed/unkey/tree/main/examples/nextjs-expiration",
```

might be a copypaste mistake here :)